pp/integrations/plugin-packs/procedures/applications-eclipse-mosquitto-mqtt.md

```diff
@@ -0,0 +1,483 @@
+| Messages      | App-Eclipse-Mosquitto-Mqtt-Messages-custom | Check messages count, open                           |
+| Uptime        | App-Eclipse-Mosquitto-Mqtt-Uptime-custom   | Time since the server has been working and available |
+
+> The services listed above are created automatically when the **App-Eclipse-Mosquitto-Mqtt-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias | Service Template                                | Service Description                                     |
+|:--------------|:------------------------------------------------|:--------------------------------------------------------|
+| Numeric-Value | App-Eclipse-Mosquitto-Mqtt-Numeric-Value-custom | Check that retrieves a digital value from a topic |
+| String-Value  | App-Eclipse-Mosquitto-Mqtt-String-Value-custom  | Check that retrieves a string value from a topic  |
+
+> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.
+
+</TabItem>
+</Tabs>
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Clients" label="Clients">
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| client.connected | count     |
+| client.maximum | count     |
+| client.active | count     |
+| client.inactive | count     |
+
+</TabItem>
+<TabItem value="Messages" label="Messages">
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| messages.stored | count     |
+| messages.received | count     |
+| messages.sent | count     |
+
+</TabItem>
+<TabItem value="Numeric-Value" label="Numeric-Value">
+
+The obtained metrics depend fully on the configuration.
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="String-Value" label="String-Value">
+
+No metrics for this service.
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Uptime" label="Uptime">
+
+| Metric name           | Unit  |
+|:----------------------|:------|
+| system.uptime.seconds | s     |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+The **Eclipse Mosquitto MQTT** monitoring connector needs network access to the **Eclipse Mosquitto MQTT** instance.
+If the **Eclipse Mosquitto MQTT** server is configured to use SSL, make sure the connector is able to validate the server's certificate.
+If the **Eclipse Mosquitto MQTT** server is configured to use authentication, make sure the connector is able to authenticate.
```

Comment:
```suggestion
If the **Eclipse Mosquitto MQTT** server is configured to use SSL, make sure the server's certificate is recognized by the poller, or that the certification authority's certificate is present on it. Otherwise, the host must be monitoring without controlling the certificate's legitimacy.
If the **Eclipse Mosquitto MQTT** server is configured to use authentication, make your have a read-only account.
```